pysilcam/silcamgui/interactive_summary.py

```diff
@@ -461,76 +500,88 @@ def toggle_plot(self):
+        psd_gor = sum(ds_psd['vd_gas'].values) / (sum(ds_psd['vd_oil'].values) + sum(ds_psd['vd_gas'].values)) * 100
 
-        plt.sca(self.axispsd)
-        plt.cla()
-        plt.plot(self.dias, psd_total, 'k', linewidth=5, label='Total')
-        plt.plot(self.dias, psd_oil, color=[0.7, 0.4, 0], label='Oil')
-        plt.plot(self.dias, psd_gas, 'b', label='Gas')
+        #plt.sca(self.axispsd)
+        self.axispsd.clear()
+        self.axispsd.plot(self.dias, ds_psd['vd_total'], 'k', linewidth=5, label='Total')
+        self.axispsd.plot(self.dias, ds_psd['vd_oil'], color=[0.7, 0.4, 0], label='Oil')
+        self.axispsd.plot(self.dias, ds_psd['vd_gas'], 'b', label='Gas')
 
-        plt.vlines(psd_d50_total, 0, max(psd_total), 'k', linestyle='--', linewidth=1, label='Total d50: {:0.0f}um'.format(psd_d50_total))
-        plt.vlines(psd_d50_oil, 0, max(psd_oil), color=[0.7, 0.4, 0], linestyle='--', linewidth=1, label='Oil d50: {:0.0f}um'.format(psd_d50_oil))
-        plt.vlines(psd_d50_gas, 0, max(psd_gas), 'b', linestyle='--', linewidth=1, label='Gas d50: {:0.0f}um'.format(psd_d50_gas))
+        self.axispsd.vlines(psd_d50_total, 0, max(ds_psd['vd_total']), 'k', linestyle='--', linewidth=1, label='Total d50: {:0.0f}um'.format(psd_d50_total))
+        self.axispsd.vlines(psd_d50_oil, 0, max(ds_psd['vd_oil']), color=[0.7, 0.4, 0], linestyle='--', linewidth=1, label='Oil d50: {:0.0f}um'.format(psd_d50_oil))
+        self.axispsd.vlines(psd_d50_gas, 0, max(ds_psd['vd_gas']), 'b', linestyle='--', linewidth=1, label='Gas d50: {:0.0f}um'.format(psd_d50_gas))
 
-        plt.vlines(psd_peak_total, 0, max(psd_total), 'k', linestyle=':', linewidth=1, label='Total peak: {:0.0f}um'.format(psd_peak_total))
-        plt.vlines(psd_peak_oil, 0, max(psd_oil), color=[0.7, 0.4, 0], linestyle=':', linewidth=1, label='Oil peak: {:0.0f}um'.format(psd_peak_oil))
-        plt.vlines(psd_peak_gas, 0, max(psd_gas), 'b', linestyle=':', linewidth=1, label='Gas peak d50: {:0.0f}um'.format(psd_peak_gas))
+        self.axispsd.vlines(psd_peak_total, 0, max(ds_psd['vd_total']), 'k', linestyle=':', linewidth=1, label='Total peak: {:0.0f}um'.format(psd_peak_total))
+        self.axispsd.vlines(psd_peak_oil, 0, max(ds_psd['vd_oil']), color=[0.7, 0.4, 0], linestyle=':', linewidth=1, label='Oil peak: {:0.0f}um'.format(psd_peak_oil))
+        self.axispsd.vlines(psd_peak_gas, 0, max(ds_psd['vd_gas']), 'b', linestyle=':', linewidth=1, label='Gas peak d50: {:0.0f}um'.format(psd_peak_gas))
 
-        plt.xlabel('ECD [um]')
-        plt.ylabel('VD [uL/L]')
-        plt.xscale('log')
-        plt.xlim(10, 12000)
-        plt.legend(loc='upper left')
+        self.axispsd.set_xlabel('ECD [um]')
+        self.axispsd.set_ylabel('VD [uL/L]')
+        self.axispsd.set_xscale('log')
+        self.axispsd.set_xlim(10, 12000)
+        self.axispsd.legend(loc='upper left')
 
-        plt.sca(self.axistext)
+        #plt.sca(self.axistext)
```

Comment:
E265: block comment should start with '# '  <!-- Automatically posted by Lintly -->